src/language-js/embed/utils.js

```diff
@@ -30,15 +30,24 @@ const angularComponentObjectExpressionPredicates = [
  * })
  */
 function isAngularComponentStyles(path) {
-  return path.match(
-    (node) => node.type === "TemplateLiteral",
-    (node, name) => isArrayOrTupleExpression(node) && name === "elements",
-    (node, name) =>
-      isObjectProperty(node) &&
-      node.key.type === "Identifier" &&
-      node.key.name === "styles" &&
-      name === "value",
-    ...angularComponentObjectExpressionPredicates,
+  const isTemplateLiteral = (node) => node.type === "TemplateLiteral";
+  const isStyleProperty = (node, name) =>
+    isObjectProperty(node) &&
+    node.key.type === "Identifier" &&
+    node.key.name === "styles" &&
```

Comment:
`!node.computed`

Reply:
I fixed this and added tests. Also I've create the new similar issue for `template` as a good first ( https://github.com/prettier/prettier/issues/15969 )